build.gradle.kts

```diff
@@ -16,15 +15,18 @@ buildscript {
   }
 
   dependencies {
-    classpath(libs.kotlinAllOpenPlugin)
-    classpath(libs.kotlinGradlePlugin)
     classpath(libs.detektGradlePlugin)
     classpath(libs.dokkaGradlePlugin)
+    classpath(libs.flywayGradlePlugin)
+    classpath(libs.jgit)
+    classpath(libs.jooqGradlePlugin)
+    classpath(libs.kotlinAllOpenPlugin)
+    classpath(libs.kotlinGradlePlugin)
     classpath(libs.kotlinNoArgPlugin)
+    classpath(libs.mysql)
     classpath(libs.protobufGradlePlugin)
-    classpath(libs.jgit)
-    classpath(libs.wireGradlePlugin)
     classpath(libs.sqldelightGradlePlugin)
+    classpath(libs.wireGradlePlugin)
```

Comment:
I also sorted this list because 🙈 

The additions are flyway, jooq, and mysql.